setup_deb.sh

```diff
@@ -98,7 +98,16 @@ pip3 install --no-deps --upgrade \
    scikit-image==0.19.2 \
    scikit-learn==1.0.2 \
    threadpoolctl==3.1.0 \
-   tokenizers==0.12.1
+   tokenizers==0.12.1 \
+   absl-py \
+   wrapt \
+   opt_einsum \
+   gast \
+   astunparse \
+   termcolor \
+   keras_preprocessing \
+   flatbuffers \
+   tensorflow==2.7.1
```

Comment:
Installing tensorflow here will override tensorflow-aio present in e.g. our docker or some dev environment with native version.

Reply:
Thats why there is sort of a hack with install_frameworks.py script

Reply:
@dkupnicki would you please have a look

Reply:
@jan-grzybek-ampere , you are correct. The pip install tensorflow should be removed. Actually, I think the simplest fix is simply moving the TF check in install_frameworks.py to be the first one.

Reply:
@dkupnicki , please investigate the CI test issue with the bert model.